pymeasure/instruments/rohdeschwarz/fsl.py

```diff
@@ -130,14 +130,25 @@ def continue_single_sweep(self):
     # Traces ------------------------------------------------------------------
 
     def read_trace(self, n_trace=1):
+
         """
         Read trace data.
```

Comment:
Maybe document that this is the trace data from the active channel.

Reply:
Resolved via [1d49a15](https://github.com/bleykauf/pymeasure/pull/4/commits/1d49a15a1c3277b3bcad59f81feb01e81f3c4836)